### ISSUE TITLE
Not really an issue but can you add windows support...

### ISSUE THREAD

**Issue by mjoe67886** [donkirkby/live-py-plugin#618]
Yeah if you could add Windows 11 support that would be awesome

**Comment by donkirkby** [comment]
Works for me with PyCharm 2024.1 on Windows 11 Home 22H2.

Please describe what happens when you try to run it, and tell me the rest of the information from the [issue template].

[issue template]: https://github.com/donkirkby/live-py-plugin/blob/master/ISSUE_TEMPLATE.md

**Comment by mjoe67886** [comment]
Thank you very much for providing this.... As long as you affirm it works
on windows I'll trust you. I haven't tried it yet but I'm really looking
forward to it....... I love Ubuntu so much I was scratching my head how I
could incorporate Ubuntu into my Blender add on which I developed
religiously... To make your system work with mine...but I couldn't figure
it out... As a thank you for giving everybody this gift which is really
awesome if you ever tried it with common lisp in Emacs.... I created the
biggest Blender add on in the world code wise it has over 4 million lines
of code. If you ever do 3D animation or modeling I'll send you a copy of my
development version. It's got tons of features... And it's really nice to
python developers.... It has over sixteen P I P libraries bundled with
it..... so as a developer you would have even more fun with this... here it
is.... fast-blender-add-ons.com If you would like to see it it's
called....FAST Animation Studio Tools....

Thanks again I was just going to write this myself the other day uh but I
thought to look online first and you had that beautiful website and you
keep it up to date so religiously so I really appreciate it again....
People don't really understand what live coding is and what it could do so
if you made a video like a more recent video for people....I would make a
ton of videos all about this.............the stuff that would be cool with
live coding when I get a chance I'm going to incorporate it into Blender's
text editor so people could use live coding in Blender Scripting...... And
that will be so awesome!......(note...Blender's text editor is the ide for
Blender)

Best Regards!
